packages/wallet-core/src/jwt-session/index.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as v from 'valibot';
+import type { JWTPayload } from 'jose';
+import { SignJWT, decodeJwt, jwtVerify } from 'jose';
+
+const AccountSchema = v.object({
+	address: v.string(),
+	publicKey: v.string(),
+	nickname: v.optional(v.string()),
+	icon: v.optional(v.string()),
+	chains: v.optional(v.array(v.pipe(v.string(), v.regex<`${string}:${string}`>(/^.+:.+$/)))),
+	features: v.optional(v.array(v.pipe(v.string(), v.regex<`${string}:${string}`>(/^.+:.+$/)))),
+});
+
+const JwtSessionSchema = v.looseObject({
```

Comment:
This is way too much info, let's trim down similar to what we did here: https://github.com/MystenLabs/wallet-next/pull/1210/files#diff-bc3253061283423c721c833f262624f2ba8b6073381c5122cc6671b500cfc743